nautobot/extras/templatetags/job_buttons.py

```diff
@@ -154,23 +158,34 @@ def job_buttons(context, obj):
                 if text_rendered:
                     template_args["button_text"] = text_rendered
                     if jb.confirmation:
-                        buttons_rendered += "<li>" + CONFIRM_BUTTON.format(**template_args) + "</li>"
-                        forms_html += CONFIRM_MODAL.format(**template_args)
+                        buttons_rendered += (
+                            mark_safe("<li>")  # noqa: S308
+                            + format_html(CONFIRM_BUTTON, **template_args)
+                            + mark_safe("</li>")  # noqa: S308
+                        )
```

Comment:
```suggestion
                        buttons_rendered += format_html(f"<li>{CONFIRM_BUTTON}</li>", **template_args)
```

Not blocking, just more concise and accomplishes the same thing?

Reply:
Probably so, good thought.